web_src/js/features/repo-home.js

```diff
@@ -3,6 +3,9 @@ import {stripTags} from '../utils.js';
 
 const {appSubUrl, csrfToken} = window.config;
 
+const {lang} = document.documentElement;
+const shortDateFormatter = new Intl.DateTimeFormat(lang, {year: 'numeric', month: 'short', day: 'numeric'});
```

Comment:
can you reuse
https://github.com/go-gitea/gitea/blob/b5b3e0714e624cea3ce4d5368aa1266f7639d0eb/web_src/js/features/formatting.js#L5
?

Maybe `export` here?